src/config/index.ts

```diff
@@ -53,7 +54,7 @@ export const FEATURE_FLAG_ENABLE_UPDATE_STATEMENT_VALIDATION = getEnvironmentVal
 export const FEATURE_FLAG_ENABLE_UPDATE_MANAGE_TRUSTS = getEnvironmentValue('FEATURE_FLAG_ENABLE_UPDATE_MANAGE_TRUSTS_29082023', 'false');
 export const FEATURE_FLAG_ENABLE_REDIS_REMOVAL = getEnvironmentValue('FEATURE_FLAG_ENABLE_REDIS_REMOVAL_27092023', 'false');
 export const FEATURE_FLAG_ENABLE_CEASE_TRUSTS = getEnvironmentValue('FEATURE_FLAG_ENABLE_CEASE_TRUSTS_19022024', 'false');
-export const FEATURE_FLAG_ENABLE_RELEVANT_PERIOD = getEnvironmentValue('FEATURE_FLAG_ENABLE_RELEVANT_PERIOD_26042024', 'false');
+export const FEATURE_FLAG_ENABLE_RELEVANT_PERIOD = getEnvironmentValue('FEATURE_FLAG_ENABLE_RELEVANT_PERIOD_26042024', 'true');
```

Comment:
it's okay to set these to true on your Local, although you shouldn't need to. It's not a good idea to commit these as true on the repo. These values are set through environment variables so that we can have different features turned on and off in different environments.

Reply:
The values you've changed are the default values. i.e the value when no environment variable is found with that name. The service defaults to false for all these as a precaution I believe. If you need to change these on your local, a better way to do it is to set the values in `overseas-entities-web.docker-compose.yaml` in your docker-chs-development repo

Reply:
> it's okay to set these to true on your Local, although you shouldn't need to. It's not a good idea to commit these as true on the repo. These values are set through environment variables so that we can have different features turned on and off in different environments.

I'll update the feature flags now.